projects/angular/src/data/datagrid/_properties.datagrid.scss

```
@@ -9,86 +9,78 @@
-      --clr-datagrid-detail-pane-close-padding: 0 0 0 #{variables.$clr_baselineRem_0_5};
-      --clr-datagrid-detail-body-padding: 0 #{variables.$clr_baselineRem_1};
-      --clr-datagrid-detail-pane-close-btn-min-width: #{variables.$clr_baselineRem_3};
-      --clr-datagrid-detail-pane-content-padding: 0;
-      --clr-datagrid-detail-pane-close-icon-size: #{variables.$clr_baselineRem_1};
-      --clr-datagrid-detail-body-text-color: #{variables.$clr-p1-color};
-      --clr-datagrid-detail-header-title-color: #{variables.$clr-p1-color};
-      --clr-datagrid-detail-pane-bg-color: var(--clr-datagrid-popover-bg-color);
-      --clr-datagrid-detail-pane-border-color: var(--clr-datagrid-default-border-color);
+    --clr-datagrid-column-toggle-border-active-color: #{tokens.$cds-alias-object-interaction-border-color};
+    --clr-datagrid-column-toggle-fill-active-color: #{tokens.$cds-alias-object-opacity-0};
+    --clr-datagrid-column-toggle-text-active-color: #{tokens.$cds-alias-typography-color-400};
 
-      --clr-datagrid-footer-padding: 0 #{variables.$clr_baselineRem_0_5};
-      --clr-datagrid-placeholder-padding: #{variables.$clr_baselineRem_0_5};
-      --clr-datagrid-placeholder-gap: #{variables.$clr_baselineRem_0_5};
-      --clr-datagrid-placeholder-font-size: #{variables.$clr-h5-font-size};
-      --clr-datagrid-placeholder-font-weight: inherit;
-      --clr-datagrid-placeholder-line-height: inherit;
-      --clr-datagrid-placeholder-letter-spacing: inherit;
-      --clr-datagrid-placeholder-background-color: #{tables-variables.$clr-table-bgcolor};
-      --clr-datagrid-column-seperator-height: #{variables.$clr_baselineRem_1_0833};
-      --clr-datagrid-filter-toggle-size: #{datagrid-variables.$clr-datagrid-icon-size};
-      --clr-datagrid-row-active-color: var(--clr-datagrid-row-hover-color);
-      --clr-datagrid-footer-line-height: #{variables.$clr_baselineRem_1_5 - (3 * variables.$clr-global-borderwidth)};
-      --clr-datagrid-footer-pagination-line-height: #{variables.$clr_baselineRem_1_5};
-      --clr-datagrid-fixed-column-size: #{datagrid-variables.$clr-datagrid-fixed-column-size};
-    }
+    --clr-datagrid-detail-pane-content-padding: #{tokens.$cds-global-space-7};
+    --clr-datagrid-detail-pane-close-icon-size: #{tokens.$cds-global-space-9};
+    --clr-datagrid-detail-body-text-color: #{tokens.$cds-alias-typography-color-400};
+    --clr-datagrid-detail-header-title-color: #{tokens.$cds-alias-typography-color-400};
+    --clr-datagrid-detail-pane-bg-color: #{tokens.$cds-alias-object-container-background};
+    --clr-datagrid-detail-pane-border-color: #{tokens.$cds-alias-object-border-color};
+
+    --clr-datagrid-placeholder-font-size: var(--clr-h4-font-size);
+    --clr-datagrid-placeholder-font-weight: var(--clr-h4-font-weight);
+    --clr-datagrid-placeholder-line-height: var(--clr-h4-line-height);
+    --clr-datagrid-placeholder-letter-spacing: var(--clr-h4-letter-spacing);
```

If I'm not mistaken typography tokens should be based on `'SECTION-20-STD'`.

```suggestion
    --clr-datagrid-placeholder-font-size: #{tokens.$cds-alias-typography-section-font-size};
    --clr-datagrid-placeholder-font-weight: #{tokens.$cds-alias-typography-section-font-weight};
    --clr-datagrid-placeholder-line-height: #{tokens.$cds-alias-typography-section-line-height};
    --clr-datagrid-placeholder-letter-spacing: #{tokens.$cds-alias-typography-section-letter-spacing};
```